metricbeat/metricbeat.full.yml

```diff
@@ -64,6 +64,11 @@ metricbeat.modules:
   # if true, exports the CPU usage in ticks, together with the percentage values
   #cpu_ticks: false
 
+  # A list of filesystem types to ignore. The filesystem metricset will not
+  # collect data from filesystems matching any of the specified types and
+  # fsstats will not include data from these filesystems in its summary stats.
+  #filesystem.ignore_types: []
```

Comment:
Would it make sense to populate the list with types that users commonly want to specify? Would be good to show as an example.